doc/shell_long/README.md

```diff
@@ -0,0 +1,891 @@
+# 1. Test Objective  
+The document is to introduce how to test shell_long via CTP. It is similiar with shell test with CTP.For some shell cases, it spend so much time which affects the end of shell regression test, then remove them to shell_long test category.    
+
+# 2. Tools Introduction  
+## 2.1 CTP tools
+Please refer to [README.md](https://github.com/CUBRID/cubrid-testtools/blob/develop/CTP/README.md) of CTP  

```

Comment:
Need inherit shell guide (provide a link).

General usage should be merged to shell guide.

Please check information provided in this guide and decide whether to merge to shell guide.